exercises/verify.cpp

```diff
@@ -9,10 +9,32 @@ The number 3 is [not] present in the array.
 #include <iostream>
 using namespace std;
 
+#define EXIT_SUCCESS 0
+
+bool search(int* vett, int n, int x){
+  bool trovato = 0;
+
+  for(int i=0;i<n;i++){
+    if(vett[i]==x){
+      trovato = 1;
+      break;
+    }
+  }
+  
+  return trovato;
+}
+
 int main()
 {
   // placeholder
   int N[10] = {3, 4, 5, 1, 2, 3, 4, 9, 13, 0};
+  int number;
+
+  cout << "Insert number: ";
+  cin >> number;
+
+  if(search(N, 10, number)==true) cout << "The number " << number << " is present in the array" << endl;
```

Comment:
puoi omettere `==true` perchè già `search()` restituisce un booleano
```suggestion
  if(search(N, 10, number)) cout << "The number " << number << " is present in the array" << endl;
```